build.sbt

```diff
@@ -22,6 +22,7 @@ scalacOptions ++= Seq(
   "-Wdead-code",                // Warn when dead code is identified.
   "-Wextra-implicit",           // Warn when more than one implicit parameter section is defined.
   "-Wunused",                   // Warn if something from check list is unused.
+  "-Wconf:msg=(Quoted|idiomContext@_)&cat=unused:s", // add exceptions for quill
```

Comment:
Just to do not waste our time we can do it in the next request, but I suggest apply formatting here